packages/react-tooltip/etc/react-tooltip.api.md

```diff
@@ -34,7 +34,7 @@ export type TooltipCommons = {
     positioning?: PositioningShorthand;
     visible?: boolean;
     onVisibleChange?: (event: React_2.PointerEvent<HTMLElement> | React_2.FocusEvent<HTMLElement> | undefined, data: OnVisibleChangeData) => void;
-    triggerAriaAttribute: 'label' | 'labelledby' | 'describedby' | null;
+    relationship: 'description' | 'label' | 'none';
```

Comment:
I would propose to change `none` to something more disturbing like `noLabelOrDescription`

Reply:
plus ensuring it is a required prop without default would make the API safer to use

Reply:
I changed it to `inaccessible`; how does that sound? (cc @smhigley)